doc/JSON_FLAGS.md

```diff
@@ -707,6 +715,7 @@ Some armor flags, such as `WATCH` and `ALARMCLOCK` are compatible with other ite
 - ```MUTAGEN_ALPHA``` Causes mutation in the alpha branch.
 - ```MUTAGEN_ELFA``` Causes mutation in the elfa branch.
 - ```MUTAGEN_RAPTOR``` Causes mutation in the raptor branch.
+- ```SERUM``` This mutagen is more powerful then its lesser versions. Used to determinate your pets mutation strength.
```

Comment:
More powerful "than", not "then". "Then" is for conditionals (if-then) or time, "than" for comparison.
Used to determine, not determinate.
Your pet's (the apostrophe)